src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -367,6 +367,17 @@ public function get_items( $request ) {
 			add_filter( 'post_password_required', array( $this, 'check_password_required' ), 10, 2 );
 		}
 
+		$parent_ids = array();
+		foreach ( $query_result as $post ) {
+			if ( $post->post_parent > 0 ) {
+				$parent_ids[] = $post->post_parent;
+			}
+		}
+
+		if ( ! empty( $parent_ids ) ) {
+			_prime_post_caches( $parent_ids, false, false );
```

Comment:
```suggestion
			_prime_post_caches( $parent_ids, false );
```

`current_user_can( 'read_post', $post_id )` calls `get_post_status()` which can call `get_post_meta()` so it's probably safer to warm the post meta anyway.